xbmc/guilib/GraphicContext.cpp

```diff
@@ -373,15 +374,15 @@ void CGraphicContext::SetVideoResolution(RESOLUTION res, bool forceUpdate)
 {
   if (g_application.IsCurrentThread())
   {
-    SetVideoResolutionInternal(res, forceUpdate);
+    SetVideoResolutionInternal(res, forceUpdate, true);
   }
   else
   {
-    CApplicationMessenger::GetInstance().SendMsg(TMSG_SETVIDEORESOLUTION, res, forceUpdate ? 1 : 0);
+    CApplicationMessenger::GetInstance().SendMsg(TMSG_SETVIDEORESOLUTION, res, forceUpdate);
   }
 }
 
-void CGraphicContext::SetVideoResolutionInternal(RESOLUTION res, bool forceUpdate)
+void CGraphicContext::SetVideoResolutionInternal(RESOLUTION res, bool forceUpdate, bool callWindowing)
```

Comment:
please drop 3rd parameter. I rather have some duplicated lines of code than trying to reuse code that does not fit the purpose.